src/routes/middleware/validators/recipe.validator.ts

```diff
@@ -79,7 +73,15 @@ const validateData = [
   preparingContent,
   ingredientsContent,
   ingredients,
-  url,
+];
+
+const validateDataForUpdate = [
+  title.optional(),
+  description,
+  preparing.optional(),
```

Comment:
hmm, I remember there was an issue with express-validator that adding `.optional` like this to the e.g. `preparing` altered `preparing` variable. Make sure it's working properly

Reply:
There was exactly this error